src/registrar/admin.py

```diff
@@ -335,10 +343,11 @@ class UserDomainRoleAdmin(ListHeaderAdmin):
     search_fields = [
         "user__first_name",
         "user__last_name",
+        "user__email",
         "domain__name",
         "role",
     ]
-    search_help_text = "Search by user, domain, or role."
+    search_help_text = "Search by firstname, lastname, email, domain, or role."
```

Comment:
**Question (non-blocking):** Would it be possible to split up `first name` and `last name`? (Not sure if this is caption search help text or if it's for accessibility readout reasons, so feel free to correct me)